### DISCUSSION TITLE
Is it possible to resize a dovi profile 7 BL+EL+RPU video?

### DISCUSSION THREAD

**Discussion by Drukajin** [quietvoid/dovi_tool#214] [Q&A] [answered]
Hello,

I've just started using dovi tools, and it's been good to use. I am having trouble resizing 4k profile 7 BL+EL+RPU files down to 1080p. I have been able to manage this with profile 8 BL+RPU, so I'm not sure if what I am asking is even possible. The end result always comes out with the wrong aspect ratio.

**Comment by quietvoid** [comment, marked as answer]
If you have cropped the letterbox bars, just use `--crop` when extracting or `"crop": true` when editing the RPU.

**Reply by Drukajin** [reply]
Okay thanks. I'll give that a go. I just realised i said it was the aspect ratio that was off. It looks more like it has been overcropped rather than the aspect ratio being off.

**Reply by quietvoid** [reply]
Yes that's what happens if you don't edit the RPU and the video was cropped.
It only does that on non-HDMI input sources, like internal players.

**Reply by Drukajin** [reply]
Thanks man, that fixed it. I am not using a command to crop the video, just to resize it to 1080p and to keep the aspect ratio. Maybe my encoder is cropping it automatically.
